cnf-tests/pod-utils/utils.go

```diff
@@ -0,0 +1,40 @@
+package pod_utils
```

Comment:
utils packages are an antipattern in Go. Also, underscore in the package name is not idiomatic.

I'd suggest to use a package with a name like `node` and rename `PrintNodeInformation` to just `PrintInformation`

more info in
https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common
https://blog.golang.org/package-names

Reply:
Also, I'd suggest to create it under a pkg subdir so it's clear the others are executables and this is a package